data/json/items/tool/musical_instruments.json

```diff
@@ -15,6 +15,7 @@
     "material": [ "wood", "iron" ],
     "symbol": "|",
     "color": "white",
+    "max_charges": 1,
```

Comment:
I want to test whether instruments actually need charges, AFAIK they don't use them, but they might need one charge for the use invocation to work.

Reply:
I just tried the violin and it looks like they don't rquire charges, or at least the violin doesn't.